scripts/setup

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+conf="
+;; Ethereum conf
+[web3]
+provider=ws://127.0.0.1:9545 ;; truffle ws"
+
+echo "${conf}" > /root/.ararc
+
+for repo in ara-identity ara-network; do
+  git clone git@github.com:AraBlocks/${repo}.git
+  if [ $? -ne 0 ]; then
+    exit 1
+  fi
+  cd ${repo}
+  npm install
+  npm link
+  cd ..
+done
+
+mkdir -p /root/.ara/keyrings
+
+# Create identity
+echo "test" | aid create
```

Comment:
what does the `echo "test"` do here?

Reply:
It sets the password of the identity to "test". :)

Reply:
ahhhh, TIL. 🙌